SPEC.md

```diff
@@ -700,8 +702,8 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
     },
     "dependencies": [
         {
-            "app": "example.com/reduce-worker-base",
-            "imageID": "sha512-...",
+            "name": "example.com/reduce-worker-base",
+            "id": "sha512-...",
```

Comment:
dropping this to ID here seems too terse. UUID is an ID as well, but that is not what is meant here. Sorry if it's off-topic, it just seems a little too terse. Like the next step would be to loosen up the requirement to be Image ID formatted.